package/src/renderer/nodes/Drawing.tsx

```diff
@@ -47,20 +46,19 @@ export class DrawingNode<P> extends Node<P> {
   }
 
   draw(ctx: DrawingContext) {
-    const drawingProps = materialize(this.props);
```

Comment:
isn't the issue in this function that `Node._props` is now `P` and not `Animated<P>`? This should enable the removing the `as P` references.

Reply:
The reasoning behind this was to keep the pattern simple, so when you set props, you are allowed to set both regular and animated versions of each prop (where Animated version is either a SkiaValue of the same type or a SkiaSelector). Then the props are resolved to values immediately (and on subsequent updates) and listeners are installed to update props whenever a SkiaValue change. 

Having the props directly mutated also ensures that we don't use double amount of memory for props in a Node.

Reply:
I see that this looks a bit wrong on the type side - we shouldn't need to cast the value like this, but it makes it a bit more efficient since we don't need to duplicate props.